### PR TITLE
Add loading animation to search modal

### PR DESCRIPTION
**Pull request by SudharakaP**
Related to the discussion; https://opencollective.slack.com/archives/C035S573ZD2/p1655114037455389

https://user-images.githubusercontent.com/12435965/173692705-db360f0d-9b8a-48bd-aa68-d3b410f2bc73.mp4

### CODE REVIEW

**Review by znarf — APPROVED** [review]
Good for me.

File: components/TopBarV2.js [hunk 1]
```diff
@@ -70,6 +71,14 @@ const TopBarV2 = ({ showSearch, menuItems, showProfileAndChangelogMenu }) => {
   const [showSearchModal, setShowSearchModal] = useState(false);
   const ref = useRef();
 
+  useEffect(() => {
+    if (showSearchModal) {
+      NProgress.configure({ parent: '.searchModal' });
```
Comment (Betree) [inline components/TopBarV2.js:76]: Could this be solved by changing the `zIndex` of NProgress? This looks quite hacky

Comment (SudharakaP) [reply components/TopBarV2.js]: @Betree : Definitely a better approach. 🤔 I've tweaked the NProgress css to handle this. 👍🏽

File: components/Search.js [hunk 2]
```diff
@@ -23,6 +24,7 @@ const SearchModal = ({ onClose }) => {
       height="128px"
       onClose={onClose}
       overflow="hidden"
+      className="searchModal"
```
Comment (Betree) [inline components/Search.js:27]: Let's remove this `className` if unused

**Pull request merged**